citations-impl/impl/src/java/org/sakaiproject/citation/impl/BaseCitationService.java

```diff
@@ -3062,6 +3072,15 @@ public void saveCitation(Citation citation)
 			save(citation);
 		}
 
+		/*
+		 * (non-Javadoc)
+		 *
+		 * @see org.sakaiproject.citation.api.CitationCollection#saveCitationCollectionOrder(org.sakaiproject.citation.api.CitationCollectionOrder)
+		 */
+		public void saveCitationCollectionOrder(CitationCollectionOrder citationCollectionOrder)
```

Comment:
This feels really weird, I see there is already a method before this, of the same pattern, but why would you ever want to save a collection order that wasn't part of this collection?